clients/drcachesim/tools/view.h

```diff
@@ -105,7 +105,7 @@ class view_t : public analysis_tool_t {
             stream << "------------------------------------------------------------\n";
         prev_tid_ = memref.instr.tid;
         prev_record_ = record_ord;
-        stream << std::setw(9) << record_ord << std::setw(9)
+        stream << std::setw(12) << record_ord << std::setw(12)
                << memstream->get_instruction_ordinal() << ": T" << memref.marker.tid
                << " ";
     }
```

Comment:
(There are also instances in the drmemtrace_samples repo but it is fine to leave them and let them naturally get updated the next time we update those sample traces.)